autoscale/src/main/java/com/sequenceiq/periscope/monitor/evaluator/ClusterStateEvaluator.java

```diff
@@ -0,0 +1,44 @@
+package com.sequenceiq.periscope.monitor.evaluator;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.periscope.monitor.context.ClusterIdEvaluatorContext;
+import com.sequenceiq.periscope.monitor.context.EvaluatorContext;
+import com.sequenceiq.periscope.monitor.event.UpdateFailedEvent;
+
+@Component("ClusterStateEvaluator")
+@Scope("prototype")
+public class ClusterStateEvaluator extends EvaluatorExecutor {
+
+    private static final String EVALUATOR_NAME = ClusterStateEvaluator.class.getName();
+
+    @Inject
+    private EventPublisher eventPublisher;
+
+    private long clusterId;
+
+    @Override
+    public void setContext(EvaluatorContext context) {
+        clusterId = (long) context.getData();
+    }
+
+    @Override
+    @Nonnull
+    public EvaluatorContext getContext() {
+        return new ClusterIdEvaluatorContext(clusterId);
+    }
+
+    @Override
+    public String getName() {
+        return EVALUATOR_NAME;
+    }
+
+    @Override
+    public void execute() {
+        eventPublisher.publishEvent(new UpdateFailedEvent(clusterId));
```

Comment:
The handler for this ends up putting a cluster in the SUSPENDED state even if it already in the SUSPENDED state, which will end up being an unnecessary DB operation every minute for STOPPED clusters. Would be useful to avoid that.

Also, this will result in all known clusters being polled every minute from Cloudbreak. I think we should introduce a more light weigh API than the on that returns all Stack details (obviously a different PR for this).

Reply:
Removed DB Hit. For CB Sync optimization I have already created this jira. https://jira.cloudera.com/browse/CB-6619